core/tests/karma.conf.ts

```diff
@@ -53,7 +55,17 @@ module.exports = function(config) {
     exclude: [
       'local_compiled_js/core/templates/dev/head/**/*-e2e.js',
       'local_compiled_js/extensions/**/protractor.js',
-      'backend_prod_files/extensions/**'
+      'backend_prod_files/extensions/**',
+      // add these back after the templateCache is repaired
```

Comment:
Here and elsewhere: please mark TODOs explicitly as TODOs, using the "TODO(vojtechjelinek):" syntax. But if possible please try not to leave TODOs -- maybe @YashJipkate can work together with you to resolve them.